src/mumble/MainWindow.cpp

```diff
@@ -1585,6 +1586,7 @@ void MainWindow::qmUser_aboutToShow() {
 		qaUserLocalMute->setEnabled(false);
 		qaUserLocalVolume->setEnabled(false);
 		qaUserLocalIgnore->setEnabled(false);
+		qaUserLocalIgnoreTTS->setEnabled(false);
```

Comment:
Maybe we only have to show this, if TTS is enabled in the first place?

Reply:
Good point, I'll fix that.

Reply:
Done